### ISSUE TITLE
Calls are interrupting on Microsoft Teams

### ISSUE THREAD

**Issue by basavyr** [zen-browser/desktop#2159]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

## Problem Description

Whenever I join a Teams meeting, after 2–3 seconds, the other people are experiencing both **audio** and **video** interruptions from my side. This is not happening on other browsers that I currently have (e.g., Safari, Arc, Brave).

I made sure my Wi-Fi is restarted (multiple restarts to the OS as well), but interruptions are still happening. Keep in mind that this is only on my side; I can hear and see everyone else in the meetings… It is just that they can't hear me.

## System

Zen is installed on my M3 Pro MacBook, with macOS Sequoia `15.0.1 (24A348)`. My current version of the browser is `1.0.1-a.10`

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.10

### What platform are you seeing the problem on?

macOS - aarch64

### Relevant log output

```shell
NA
```

**Comment by mr-cheffy** [comment]
Does this happen also in firefox?

**Comment by basavyr** [comment]
No. only on Zen

**Comment by harryheres** [comment]
Also, has this happened to you on previous MacOS versions?

**Comment by basavyr** [comment]
I have started using Zen Browser with the version `15.0.1`. I now updated my macOS to `15.1 (24B83)`, and I still experience the same issue. In the meantime, Zen is also updated to `1.0.1-a.14`

**Comment by technicallypete** [comment]
This is legit. Same issue here. When in a meeting of many people, I get the "Hold on..." message every minute or so, and call resumes after a few seconds.

Since I use Zen as my "admin" and comms browser, not able to use Teams makes my use case kinda... useless.

And yes, confirmed: Firefox, etc. works fine.

FWIW, Teams is trash. It cant even load without crashing browser tab on Arc.

**Comment by basavyr** [comment]
Yeah. Unfortunately, my job relies on using Teams, so that is a must. This is why I had to stop using Zen for the moment, which is truly unfortunate.

**Comment by salamijason** [comment]
Hey it's been about a month since, are you guys still getting the issue?

**Comment by basavyr** [comment]
@salamijason If you check their [public roadmap](https://github.com/orgs/zen-browser/projects/4) you can see that this issue is still under "Verification", and the priority is set to low. They are probably focusing on more urgent bugs right now, so I guess we will have to wait a bit more

**Comment by salamijason** [comment]
Yes we've tried to verify it but not many people on the testing team uses Microsoft Teams. Was just wondering if it was still a issue so we could have another go at testing.

**Comment by adminy** [comment]
I can confirm its fine on linux. was broken at some point but now it seems to be fixed.

**Comment by JSGRD22** [comment]
@basavyr has the issue been resolved?

**Comment by salamijason** [comment]
looks like one user has said it's been fixed. for the sake of keeping issues clean, ill be closing this issue. if anyone still has this problem, please reopen.

**Comment by jammsen** [comment]
@salamijason Please re-open the issue.

I have the same issue on zen 1.11 until the current 1.12.6b - Windows 11 - I have the same interruption problems, and also another one randomly. People think i disconnect the call meanwhile i hear them clearly and after like 40 seconds my call just disappears in my browser window. Feels like a disconnection/connection issue. Im not a WLAN/Wifi - Im on 10Gbit Ethernet.
